systest/Cargo.toml

```diff
@@ -8,9 +8,9 @@ name = "systest"
 version = "0.1.0"
 
 [dependencies]
-rust_sodium-sys = { path = "../rust_sodium-sys", version = "~0.5.0" }
-libc = "~0.2.28"
+rust_sodium-sys = { path = "../rust_sodium-sys", version = "~0.6.0" }
+libc = "~0.2.33"
 
 [build-dependencies]
-ctest = "~0.1.4"
+ctest = "~0.1.6"
```

Comment:
Actually, if this was changed to `ctest = "=0.1.4"`, we could re-enable the systest in Travis (which was disabled in [this commit](https://github.com/Fraser999/rust_sodium/commit/1d9b45a2c784424d23b2b0c61c597fa44923a1f0)) due to [this bug in ctest](https://github.com/alexcrichton/ctest/issues/23).